eq/server/config/display.cpp

```diff
@@ -49,6 +49,12 @@ void _choosePixelViewport( Window& window )
     if( width == fabric::UNDEFINED && height == fabric::UNDEFINED )
     {
         window.setViewport( Viewport( .25f, .2f, .5f, .5f ));
+
+        // Make the window aspect ratio 16:10, assuming square pixels
```

Comment:
document about positioning? why width/2 as x? for single screen setup this will end up starting in the middle of the screen?